guardrails/cli/configure.py

```diff
@@ -64,8 +59,27 @@ def configure(
         help="Clear the existing token from the configuration file.",
     ),
 ):
-    if clear_token is True:
+    existing_token = _get_default_token()
+    last4 = existing_token[-4:] if existing_token else ""
+
+    if not clear_token:
+        console.print("\nEnter API Key below", style="bold", end=" ")
+        last4 and console.print(
+            "[dim]leave empty if you want to keep existing token[/dim]",
+            style="italic",
+            end=" ",
+        )
+        last4 and console.print(f"[{last4}]", style="italic")
```

Comment:
I have mixed feelings about using short-circuiting for this.  Since we're checking last4 twice in a row AND it's a bit of an odd pattern, I feel like maybe it might be more Pythonic to do:
```python
if last4:
    console.print(...)
    console.print(...)
```

Thoughts?

Reply:
very true, I don't think it's quite readable either, will change 👍🏼

Reply:
addressed!